### PR TITLE
Add com.interversehq.qview

### PR DESCRIPTION
**Pull request by Justinzobel**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.interversehq.qview

**Comment by Justinzobel** [comment]
@LeandroStanger thanks for the build attempt, this should fix the appdata issue.

**Comment by nedrichards** [comment]
bot, build com.interversehq.qview

@Justinzobel you can use the command too btw.

**Comment by barthalion** [comment]
Buildbot fails to find the icon due to this line: https://github.com/jurplel/qView/blob/master/dist/linux/qView.desktop#L6
It needs to match the app id.

File: com.interversehq.qview.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+command: qview
+rename-desktop-file: qView.desktop
+rename-icon: qview
+rename-appdata-file: qview.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --share=network
```
Comment (barthalion) [inline com.interversehq.qview.yml:14]: Why does it need network access?

File: com.interversehq.qview.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+app-id: com.interversehq.qview
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qview
+rename-desktop-file: qView.desktop
```
Comment (barthalion) [inline com.interversehq.qview.yml:6]: Ideally these renames should happen in your repository. As per freedesktop specs, icons, desktop launchers and appdata files should follow reverse DNS notation, just nothing enforces it.

File: com.interversehq.qview.yml (hunk 1 above)
Comment (jurplel) [reply com.interversehq.qview.yml:14]: Open URL function and update checking

File: com.interversehq.qview.yml (hunk 2 above)
Comment (Justinzobel) [reply com.interversehq.qview.yml]: This should now be resolved, can you please attempt a new build, thanks. :)

**Comment by jurplel** [comment]
bot, build com.interversehq.qview

**Comment by Justinzobel** [comment]
Build appears stuck in download sources phase.

**Comment by barthalion** [comment]
bot, build com.interversehq.qview

**Comment by barthalion** [comment]
bot, build com.interversehq.qview

**Comment by barthalion** [comment]
OK, fixed that weird x86_64 build failure, somehow runtime got corrupted on the worker.

bot, build com.interversehq.qView

File: com.interversehq.qview.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+app-id: com.interversehq.qView
```
Comment (barthalion) [inline com.interversehq.qview.yml:1]: The file name needs to be the same as the app ID.

**Review by barthalion — CHANGES_REQUESTED** [review]

File: com.interversehq.qview.yml (hunk 3 above)
Comment (Justinzobel) [reply com.interversehq.qview.yml]: Fixed.

**Comment by Justinzobel** [comment]
Fixed, apologies, didn't read the issue properly, was just looking at the build logs.

**Comment by Justinzobel** [comment]
bot, build com.interversehq.qView

**Comment by Justinzobel** [comment]
Flathub needs some basic linting tools that users can download and do these offline so we can iterate faster. Why is it failing now? Are you able to please shed some light on it @barthalion?

**Comment by Justinzobel** [comment]
bot, build com.interversehq.qView

**Comment by Justinzobel** [comment]
bot, build com.interversehq.qView

**Comment by Justinzobel** [comment]
bot, build com.interversehq.qView

**Comment by Justinzobel** [comment]
bot, build com.interversehq.qView

**Comment by Justinzobel** [comment]
OK what are the next steps @barthalion?

**Pull request closed without merging**